api/migrations/versions/12fcede9ef5c_adding_chat_color_theme.py

```diff
@@ -0,0 +1,35 @@
+"""adding chat_color_theme
+
+Revision ID: 12fcede9ef5c
+Revises: 5fda94355fce
+Create Date: 2024-05-26 05:51:41.171154
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+import models as models
+
+# revision identifiers, used by Alembic.
+revision = '12fcede9ef5c'
+down_revision = '5fda94355fce'
```

Comment:
hello, pls remember to rebase to migrations on main branch : )

Reply:
Hi, Yeuoly. I rebased onto dev because it already had the last revision of migrations of main, and to avoid some merge conflicts. @crazywoola I created a new migration, fyi. 
```
revision = '8d5b4339e6a0'
down_revision = '64a70a7aab8b'
```